common/test/services/prismic/opening-times.test.ts

```diff
@@ -510,6 +512,107 @@ describe('opening-times', () => {
     });
   });
 
+  describe.only('getUpcomingExceptionalPeriods', () => {
+    const exceptionalPeriods = [
+      [
+        {
+          overrideDate: london('2021-12-29'),
+          overrideType: 'Christmas and New Year' as OverrideType,
```

Comment:
nit: I wonder if you could bin these type hints by declaring a type on `exceptionalPeriods`? i.e.

```typescript
const exceptionalPeriods: ExceptionalOpeningHoursDay[]
```

I think that looks slightly cleaner and you'd get it type checking the whole array, but not especially urgent.